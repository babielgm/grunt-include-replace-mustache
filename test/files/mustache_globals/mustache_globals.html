@@include('mustache.html',{"name" : "Johnny", "single" : true})