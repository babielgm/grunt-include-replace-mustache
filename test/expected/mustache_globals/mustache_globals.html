Name: Johnny
Age: 18

Status: Single